homeassistant/components/insteon/insteon_entity.py

```diff
@@ -116,3 +121,7 @@ def _get_label(self):
             else:
                 label = f"Group {self.group:d}"
         return label
+
+    def _async_add_default_links(self):
+        """Add default links between the device and the modem."""
+        self.hass.async_add_job(self._insteon_device.async_add_default_links)
```

Comment:
```suggestion
        self.hass.async_add_executor_job(self._insteon_device.async_add_default_links)
```

`async_add_job` is legacy

Reply:
This should just be awaited. So I made the following changes:
insteon/utils.py line 237:
```
    @callback
    def async_add_default_links(service):
```
insteon/utils.py line 242:
`        async_dispatcher_send(hass, signal)`

insteon/insteon_entity.py line 125:
```
    async def _async_add_default_links(self):
        """Add default links between the device and the modem."""
        await self._insteon_device.async_add_default_links(self.address)
```

Reply:
Makes sense now :)